packages/webapp/src/App.tsx

```diff
@@ -55,6 +55,12 @@ const App = () => {
     const SentryRoutes = Sentry.withSentryReactRouterV6Routing(Routes);
     const env = useStore((state) => state.cookieValue);
     const signout = useSignout();
+    const setShowGettingStarted = useStore((state) => state.setShowGettingStarted);
+    const showGettingStarted = useStore((state) => state.showGettingStarted);
+
+    useEffect(() => {
+        setShowGettingStarted(env === 'dev' && (isCloud() || isLocal()));
```

Comment:
why a useeffect here? What issue were you seeing?

Reply:
It's just to setup the global state so we don't have to rewrite the condition multiple times, and keep it in sync with environment change. would you have done something differently?

Reply:
ah ok gotcha -- to keep things DRY. Woks for me.